src/utils/time.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Formats a time range in the format "HH[:MM]? AM|PM - HH[:MM]? AM|PM".
+ *
+ * @param startTime - The start time to parse
+ * @param endTime - The end time to parse
+ */
+export function formatTime(startTime: string, endTime: string): string {
+  const formatHour = (startTime: string): string => {
+    const [hour, minute] = startTime.split(':');
+
+    const hourNum = parseInt(hour);
+    const minuteNum = parseInt(minute);
+
+    let formattedTime = `${hourNum === 12 ? hourNum : hourNum % 12}`;
```

Comment:
startTime and endTime are 24h right? So should this be `hourNum <= 12 ? hourNum : hourNum % 12` ?

Reply:
This sounds like there is an npm library that can help us do this!

Reply:
@richardyjtian Oh yeah those should be the same I think? If `hourNum < 12` then this returns `hourNum % 12` which equals `hourNum` but thinking about it more I like your suggestion because it's clearer

@ericrkuo Yeah, I thought date-fns would be good for this and I played around with it but because `startTime` and `endTime` are strings and not Date objects, we would need to do more work to convert the strings to Dates first before we can pass it to other libraries. So it might make this function more complicated if we do

Reply:
Ohh I see, makes sense sounds good. Awesome!